docs/guides/Guide-Basic-Example.md

```diff
@@ -0,0 +1,235 @@
+# Hello Mobile Navigation
+
+Let's use React Navigation to build a simple chat application for Android and iOS.
```

Comment:
I know this is probably stupid, but maybe say something like `simple chat-like application` is better, to make sure they won't expect an actual chat to be done in the tutorial XD